scripts/proxysql/proxysql.cnf

```diff
@@ -0,0 +1,101 @@
+#file proxysql.cfg
+
+datadir="/var/lib/proxysql"
+
+admin_variables=
+{
+  mysql_ifaces="0.0.0.0:6032"
+  admin_credentials="admin:password;remote-admin:password"
+}
+
+mysql_servers =
+(
+  {
+    address="mysql-1"
+    port=3306
+    hostgroup=0
+    max_connections=200
+  },
+  {
+    address="mysql-2"
+    port=3306
+    hostgroup=1
+    max_connections=200
+  }
+)
+
+mysql_variables=
+{
+  threads=8
+  max_connections=100000
+  interfaces="0.0.0.0:3306"
+  server_version="5.7.18-proxysql"
+  connect_timeout_server=25000
+  connect_timeout_server_max=25000
+  default_charset="utf8mb4"
+  free_connections_pct=100
+  connection_warming=true
+  max_allowed_packet=16777216
+  monitor_enabled=false
+  query_retries_on_failure=0
+  shun_on_failures=999999
+  shun_recovery_time_sec=0
+  throttle_connections_per_sec_to_hostgroup=30
+}
+
+# defines all the MySQL users
+mysql_users:
+(
+  {
+    username = "root"
+    password = "password"
+    default_hostgroup = 0
+    max_connections=1000
+    active = 1
+  },
```

Comment:
I kept it as there are some proxysql tests that depend on it (to be added in the LHM-retry pr). Moreover, I like the idea of having it while having a distinction between root and writer. This can be removed if it is found to be too redundant.